packages/blocks/src/code-block/styles.ts

```diff
@@ -61,10 +70,10 @@ export const codeBlockStyles = css`
 
   #line-numbers {
```

Comment:
The font-size and line-height of #line-number should be aligned with design.

Reply:
Should I use a hardcoded value (15.6px) instead of `--affine-line-height`? Also, I think that 12px for the font size is too small what do you think? cc. @Svaney-ssman

Reply:
Seems that the fonts and line height of the code are still not aligned with design side, need @donteatfriedrice to help on updating [the design theme package](https://github.com/toeverything/blocksuite/issues/7099), after that the list line-height will be the same as code line-height